proto/beacon/p2p/v1/types.pb.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// source: proto/beacon/p2p/v1/types.proto
+// source: types.proto
```

Comment:
What did you use to generate this? Just wondering since your method seems to have carried over some comments from the proto file and that might be helpful for downstream debugging.

Reply:
I used `protoc` like this...

```
cd proto/beacon/p2p/v1
protoc --go_out=. types.proto 
```